tiny_skia/Cargo.toml

```diff
@@ -14,6 +14,6 @@ anyhow = "1.0.69"
 floem-peniko = "0.1.0"
 swash = "0.1.8"
 floem_renderer = { path = "../renderer", version = "0.1.0" }
-softbuffer = "0.3.1"
+softbuffer = "0.3.4"
```

Comment:
what exactly do you mean by "your program"

Reply:
> and should hopefully unblock me from building Lapce on RISC-V :D

You are trying to use floem somewhere or do you just casually build libraries?

Reply:
ok, nevermind, my mind just skipped the `Lapce` part

Reply:
It should be updated in Lapce lockfile, not in Floem

Reply:
why?

Reply:
Because you're trying to build Lapce, and not Floem?

Reply:
but wouldn't it make sense to patch it in floem so that in case another app uses it it'll just get the updated one so "all projects" that might use floem benefit?

Reply:
No, this manifest only tells that it needs at least `0.3.1`, it's up to the downstreams to use whichever versions they want

Reply:
Would you be fine with 3.2 so anything using floem would at least by default be able to compile on RISC-V?

Reply:
I don't know what default are you talking about
When adding floem as dependency it will use the highest version of library
```
mkdir -p floem-test
cd floem-test
cargo init
cargo add floem
grep 'softbuffer' -C 1 Cargo.lock
 "resvg",
 "softbuffer",
 "swash",
--
[[package]]
name = "softbuffer"
version = "0.3.4"
```